chef/cookbooks/hyperv/attributes/default.rb

```diff
@@ -89,6 +82,6 @@
 default[:service][:neutron][:name] = "neutron-hyperv-agent"
 default[:service][:neutron][:displayname] = "OpenStack Neutron Hyper-V Agent Service"
 default[:service][:neutron][:description] = "Service Wrapper for Openstack Neutron Hyper-V Agent"
-default[:service][:ceilometer][:name] = "ceilometer-agent-notification"
-default[:service][:ceilometer][:displayname] = "OpenStack Ceilometer Agent Notification Service"
-default[:service][:ceilometer][:description] = "Service Wrapper for Openstack Ceilometer Agent Notification"
+default[:service][:ceilometer][:name] = "ceilometer-agent-compute"
+default[:service][:ceilometer][:displayname] = "OpenStack Ceilometer Agent Compute Service"
+default[:service][:ceilometer][:description] = "Service Wrapper for Openstack Ceilometer Agent Compute"
```

Comment:
Metrics/LineLength: Line is too long. [103/100] (https://github.com/bbatsov/ruby-style-guide#80-character-limits)